api/templates/design.go

```diff
@@ -487,6 +487,27 @@ var ServiceIndex = Type("ServiceIndex", func() {
 	Required("api", "documentation")
 })
 
+var Author = Type("Author", func() {
+	Description("Author of template or repository.")
+	Attribute("name", String, "Name of the author.", func() {
+		MinLength(1)
```

Comment:
Pridal som `Author` do `repository` a `template` .... template to zatial bude dedit od repository, kde to pridame aj interne.